ci/dockerfiles/autoscaler-devbox/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM jetpackio/devbox@sha256:430afb5a5c9b4ec9f56776e8fcb82fe67fa5fabba3ba67ebb1c13b17d0e8de8d
+
+
+# Installing your devbox project
+WORKDIR /code
+USER root:root
+RUN mkdir -p /code && chown "${DEVBOX_USER}:${DEVBOX_USER}" /code
+USER ${DEVBOX_USER}:${DEVBOX_USER}
+COPY --chown=${DEVBOX_USER}:${DEVBOX_USER} devbox.json devbox.json
+COPY --chown=${DEVBOX_USER}:${DEVBOX_USER} devbox.lock devbox.lock
+
+
+# Step 6: Copying local flakes directories
```

Comment:
```suggestion
# Copying local flakes directories
```

Reply:
I wasn't aware that this `Dockerfile` was genrated via `devbox generate dockerfile` and thought that `# Step 6:` comes from copy & pasting stuff but seems like this is an upstream problem 👀 


https://github.com/jetify-com/devbox/blob/99c7921b8c247c19b6b14d9035d96fc51fc544be/internal/devbox/generate/tmpl/dev.Dockerfile.tmpl#L19

Reply:
Yep: https://github.com/jetify-com/devbox/pull/2073